os_command_injection/nithishareddy_ASSS-Project/server.js

```
@@ -0,0 +1,85 @@
+  });
+});
+app.get('/', (req, res) => {
+  const userInput = req.query.input;
+  res.send('OK');
+});
+// Vulnerable endpoint to store comments
+app.get('/comment', (req, res) => {
+  const userInput = req.query.input;
+
+  // Vulnerable eval usage (DO NOT USE IN PRODUCTION)
+  eval(`var injectedComment = '${userInput}'`);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)